source/walkthroughs/deploy/install_language_runtime/_nodejs.md.erb

```diff
@@ -37,15 +43,21 @@ Before you can deploy your app on the production server, you need to install Nod
     <td>
       <p>Run the following commands to enable <a href="https://fedoraproject.org/wiki/EPEL">EPEL</a> and to install Node.js from the <a href="https://nodesource.com/">NodeSource</a> YUM repository. With these commands, we also install a compiler so that we are able to install native addons from npm.</p>
 
-      <pre class="highlight<% if language_type == :meteor %> meteor old<% end %>"><span class="prompt">$ </span>sudo yum install -y epel-release curl
+      <pre class="highlight<% if language_type == :meteor %> meteor v13<% end %>"><span class="prompt">$ </span>sudo yum install -y epel-release curl
 <span class="prompt">$ </span>curl --fail -sSL -o setup-nodejs https://rpm.nodesource.com/setup_<% if language_type == :nodejs %>6.x<% end %><% if language_type == :meteor %>0.10<% end %>
 <span class="prompt">$ </span>sudo bash setup-nodejs
 <span class="prompt">$ </span>sudo yum install -y nodejs gcc-c++ make</pre>
 <% if language_type == :meteor %>
-<pre class="highlight meteor new"><span class="prompt">$ </span>sudo yum install -y epel-release curl
+<pre class="highlight meteor v14"><span class="prompt">$ </span>sudo yum install -y epel-release curl
 <span class="prompt">$ </span>curl --fail -sSL -o setup-nodejs https://rpm.nodesource.com/setup_4.x
 <span class="prompt">$ </span>sudo bash setup-nodejs
 <span class="prompt">$ </span>sudo yum install -y nodejs gcc-c++ make</pre>
+<% end %>
+<% if language_type == :meteor %>
+<pre class="highlight meteor v16"><span class="prompt">$ </span>sudo yum install -y epel-release curl
+<span class="prompt">$ </span>curl --fail -sSL -o setup-nodejs https://rpm.nodesource.com/setup_8.x
+<span class="prompt">$ </span>sudo bash setup-nodejs
+<span class="prompt">$ </span>sudo yum install -y nodejs gcc-c++ make</pre>
```

Comment:
As JavaScript and Meteor itself are in active development phase, this instructions might went outdated. To prevent this we should recommend to check used version of Node and NPM on building machine:

```shell
meteor npm -v
meteor node -v
```

Having the same reason we should point to official [meteor](https://guide.meteor.com/deployment.html#custom-deployment) and [nodejs](https://nodejs.org/en/download/package-manager/) installation docs.

It's also important to have right NPM version, which is conditional in dependence to Meteor's version.
```shell
# Update command should be executed after nodejs is installed
# meteor >= 1.6
npm install -g npm@5

# meteor <= 1.5
npm install -g npm@4

# meteor <= 1.4.3
npm install -g npm@3

# For meteor <= 1.3.* this is not required
```

Reply:
I think npm version issue is already covered due to the default npm versions being shipped with node versions, or am I missing something? The [official guide does not mention npm versions](https://guide.meteor.com/deployment.html#custom-deployment) either.

As for keeping this up to date, I think it is fairly safe to assume that meteor is not going to switch a major node version before node 10 achieves LTS, so based on that kind of update frequency, we should still be able to update these instructions manually, so that we don't overwhelm people with directions to check other sites.

Although, come to think of it, I think we can make a note that says keeping up to date with the major version is recommended. So for instance if meteor node version is 8.8, it is safe and recommended to update to latest 8.x (8.9.1 as of today), and likewise with major npm version.

Finally, installing nvm globally is tricker depending on how node/npm were installed in the first place. One might (and will probably) need to prepend `sudo -H` before the `npm i -g` commands.

This is a tricky topic all in all, so we should find a balanced way that @OnixGH thinks is in line with their overall documentation strategy.

Reply:
>  due to the default npm versions being shipped with node versions

None of the Node.js releases recommended for particular Meteor.js versions comes with right NPM release.

> based on that kind of update frequency, we should still be able to update these instructions manually

Well, this PR is ongoing for a weeks already :)

>  8.8, it is safe and recommended to update to latest 8.x (8.9.1 as of today), and likewise with major npm version.

Not agree, this is what we should stick to today:
```shell
$ meteor npm -v
5.5.1
$ meteor node -v
v8.8.1
```

> Finally, installing nvm globally is tricker depending on how node/npm were installed in the first place.

Not sure where from `nvm` topic came up, I haven't mentioned it before.
TL;TR; I'll never ever recommend to use `nvm` on production.

+1 looking for @OnixGH opinion.